src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -420,6 +420,7 @@ def load_arguments(self, _):
         c.argument('location', arg_type=get_location_type(self.cli_ctx))
         c.argument('launch_browser', help="Launch the created app using the default browser", default=False, action='store_true', options_list=['--launch-browser', '-b'])
         c.argument('logs', help="Configure default logging required to enable viewing log stream immediately after launching the webapp", default=False, action='store_true')
+        c.argument('html', help="Ignore app detection and deploy as an html app", default=False, action='store_true')
```

Comment:
Is it a bool type? Do we need a "arg_type=" to restrict the value?

Reply:
Not needed, we are just detecting if the flag is used or not (see 'logs' and 'launch_browser' argument above)